src/Interface/Modules/Render/ES/SRInterface.cc

```diff
@@ -564,9 +582,8 @@ namespace SCIRun {
       mCore.execute(0, 50);
```

Comment:
After merging master, this core execution sometimes causes the scene to disappear for a moment if you shift click on viewscene. My guess is that it's from the downstream screenshot changes.